app.yaml

```diff
@@ -0,0 +1,7 @@
+runtime: python38
+instance_class: F1
+entrypoint: python -m app.wsgi --port 8080
+
```

Comment:
```suggestion
automatic_scaling:
  max_instances: 1
```

Reply:
The free tier gives 28 instance hours a day, but by default it will scale based on load. This ensures that no matter what happens this app won't go beyond 1 instance

Reply:
Good call! I am also not able to become 100% certain that I _am_ on the free tier. I mightve paid GCP for something or other a long time ago and am no longer eligible. @NiallRees, you seem to know a little more about GCP than I, do you know of a way to check?

If I am not eligible, does anyone else in @sqlfluff/sqlfluff-maintainers have a fresh GCP account? Alternatively, we can:

1. Try pythonanywhere or heliohosting
2. I've implemented my second most traffic heavy app locally, so i expect my heroku hours to drop a lot; which means there's room for sqfluff online.

Reply:
Everyone gets a free 28 hours of F1 a day from the looks of things. So we should be cool. But wise to set an alert on spend > $0 so that you immediately find out if something goes weird.

Reply:
Perfect, I already have said alert set up!